### DISCUSSION TITLE
请教关于增量图计算的细节

### DISCUSSION THREAD

**Discussion by TerryWang1208** [alibaba/GraphScope#1718] [Q&A] [answered]
您好，
在一些文章中有看到graphscope“支持自动增量化处理动态图更新，请问
1. 这里的“动态图更新”是既支持图里面新增点或边的种类，也支持新增已有种类的点或边？
2. graphscope是怎样做增量化处理的呢？
谢谢！

**Comment by yecol** [comment, marked as answer]
Hi @TerryWang1208 , thank you for your interest in GraphScope!
- Here `dynamic graph` means appending graph without modifying the graph schema. e.g., please check out the example in [`libgrape-lite`](https://github.com/alibaba/libgrape-lite/tree/master/examples/gnn_sampler#sampling-on-dynamic-graph-append-only)
- for the incremental part, 
   - you can provide an incremental version of a single machine algorithm as IncEval, ref. [paper](https://homepages.inf.ed.ac.uk/wenfei/papers/sigmod17-GRAPE.pdf);
   - we also introduced some new technologies, ref. [paper](http://vldb.org/pvldb/vol14/p1613-gong.pdf)

**Comment by zhangzq22** [comment]
@TerryWang1208 ，您好，
对于graphscope“支持自动增量化处理动态图更新”,我也有一些疑问与思考，可以交流一下吗？
